packages/codemirror-graphql/src/hint.js

```diff
@@ -80,3 +81,17 @@ CodeMirror.registerHelper('hint', 'graphql', (editor, options) => {
 
   return results;
 });
+
+function getType(schema, name) {
+  if (name.slice(-1) === '!') {
+    const nestedType = getType(schema, name.slice(0, -1));
+    if (isNullableType(nestedType)) return new GraphQLNonNull(nestedType);
+    return;
```

Comment:
Why is this not `return nestedType;`? In the line above you are returning if it is nullable.

Reply:
It's because I wanted `undefined` to be returned if something like `foo!!` is passed into `name` parameter.